src/main/resources/public/js/app.js

```diff
@@ -64,6 +64,21 @@ function fire(row, column) {
         displayGameState(currModel);
         gameModel = currModel;
 
+        setTimeout( function(){
```

Comment:
It won't update the final hit box to Red before the Game Over alert is thrown. I'm not sure why it does it, but having this gives it that moment to update.